finland/router-config.json

```diff
@@ -62,6 +68,12 @@
         "walk": {
           "speed": 1.2,
           "reluctance": 1.8
+        },
+        "accessEgress": {
+          "maxStopCountForMode": {
+            "BIKE": 0,
+            "CAR": 0
+          }
```

Comment:
I don't think this is needed since I think the transfer cache requests are based on the defaults.

Reply:
As we discussed, is it ok to leave this as is? I could try to figure out why OTP does this and create a PR